### ISSUE TITLE
Working Example with a Userbase and CRUD tasks

### ISSUE THREAD

**Issue by Kentoseth** [ianwalter/bouncer#16]
It would be nice to see this library being used with a working(complete) example.

Something like a Todo App that works via a REST API only, where users can signup/login (maybe ComeOnIn might be useful for that?) and user-sessions are then managed with Bouncer for CRUD tasks.

This type of example will make it a lot clearer on how to use this library, which doesn't seem that clear at that moment (to me).

**Comment by ianwalter** [comment]
Agreed. I will get to this at some point in the future but also welcome others to take this on. Thanks!

**Comment by Kentoseth** [comment]
@ianwalter 

I can go as far as getting a Todo CRUD app going, but I am not sure what type of User model you can recommend?

If you can provide some advice on whether we should use ComeOnIn OR go vanilla for the user-creation, I can create it the app up to that point, where (beyond that) you can step in.

Let me know :)

**Comment by ianwalter** [comment]
@Kentoseth Comeonin is a password hashing library so I'm not sure what you mean by `user-creation`. The example in the README uses Comeonin to check a given password against the password hash stored in a database (fetched by Ecto). I would use the newest versions of Phoenix/Ecto for this example app.

**Comment by Kentoseth** [comment]
@ianwalter 

Sorry about that. I must have mistaken that library for another one. I'll keep investigating to find a reasonable user-management library, or perhaps a DIY (meaning us/me building it) solution may help here.
